search/utm_mgrs_coords_match.cpp

```diff
@@ -0,0 +1,251 @@
+#include "search/utm_mgrs_coords_match.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "platform/utm_mgrs_utils.hpp"
+
+#include <iterator>
+#include <string>
+#include <utility>
+#include <iostream>
+#include <math.h>
+
+using namespace std;
```

Comment:
Put this using inside namespace search. Or don't use using at all.

Reply:
Moved into namespace.